src/menu/InputEditor.cpp

```diff
@@ -51,10 +51,10 @@ void InputEditor::DrawButton(const char* label, int32_t n64Btn, int32_t currentP
         }
     }
 
-    const std::string BtnName = backend->GetButtonName(currentPort, n64Btn);
+    const std::string btnName = backend->GetButtonName(currentPort, n64Btn);
```

Comment:
another place where i'd want `button` instead of `btn`

Reply:
Out of scope.